util.py

```diff
@@ -55,6 +55,7 @@ def copyFilesFromDir(sourcedir, dest):
 
 def runCmd2(command, with_stdout=False, with_stderr=False, inputtext=None):
 
+    logger.log("running %s" % str(command))
```

Comment:
I think this might result in unnecessarily verbose logs - logging the command twice (before and after).

How about using a `try...except` and in the exception handler logging something like `Running {command} failed with exception {exception}` and then reraising the exception.